content/docs/reference/builder-config.md

```diff
@@ -5,92 +5,107 @@ weight=9
-          appear in multiple groups at once but never in the same group.
-        
-        - **`version`** _(string, optional, default: inferred)_
-          <br>
-          The version of the buildpack being referred to. This field may be omitted if the top-level `buildpacks` list contains
-          only one version of the buildpack.
-        
-        - **`optional`** _(boolean, optional, default: `false`)_
-          <br>
-          Whether or not this buildpack is optional during detection.
- 
-- **`stack`** _(required)_
+
+    - **`id`** _(string, required)_
+      <br>
+
+      The identifier of a buildpack from the configuration's top-level `buildpacks` list. Buildpacks with the same ID may
+      appear in multiple groups at once but never in the same group.
+
+    - **`version`** _(string, optional, default: inferred)_
+      <br>
+
+      The version of the buildpack being referred to. This field may be omitted if the top-level `buildpacks` list contains
+      only one version of the buildpack.
+
+    - **`optional`** _(boolean, optional, default: `false`)_
+      <br>
+
+      Whether or not this buildpack is optional during detection.
+
+- #### `stack` _(required)_
   <br>
+
   The stack to use for the builder. See [Working with stacks](/docs/concepts/components/stack) for more information about this field. It
   contains the following fields:
-  
+
   - **`id`** _(required, string)_
     <br>
+
     Identifier for the stack
-  
+
   - **`build-image`** _(required, string)_
     <br>
+
     Build image for the stack
-  
+
   - **`run-image`** _(required, string)_
     <br>
+
     Run image for the stack
-  
+
   - **`run-image-mirrors`** _(optional, string list)_
     <br>
+
     [Run image mirrors](/docs/concepts/components/stack#run-image-mirrors) for the stack
 
-- **`lifecycle`** _(optional)_
+- #### `lifecycle` _(optional)_
   <br>
+
   The [lifecycle](#lifecycle) to embed into the builder. It must contain **at most one** the following fields:
-  
+
   - **`version`** _(string, optional)_
     <br>
+
     The version of the lifecycle (semver format) to download. If specified, `uri` must not be provided.
-   
+
   - **`uri`** _(string, optional)_
     <br>
-     A URL or path to an [archive](#supported-archives). If specified, `version` must not be provided.
+
+    A URL or path to an [archive](#supported-archives). If specified, `version` must not be provided.
 
     > If `version` and `uri` are both omitted, `lifecycle` defaults to the version that was last released
     > at the time of `pack`'s release. In other words, for a particular version of `pack`, this default
-    > will not change despite new lifecycle versions being released.
-  
-#### Supported archives
+    > will not change despite new lifecycle versions being released. The current default lifecycle versin can be [seen here](https://github.com/buildpacks/pack/blob/e4315be24c103e3c5722b08561bf13a55876cbbc/internal/builder/lifecycle.go#L18).
```

Comment:
This doesn't look accurate since it's pointing to a very specific commit. Plus pointing to source code is not ideal. 

I'd rather we omit this for now and instead use the same strategy for pack version and inject it into the documentation. _This should be a separate issue._